WeScan/Scan/ScannerViewController.swift

```diff
@@ -104,11 +104,14 @@ final class ScannerViewController: UIViewController {
         navigationController?.navigationBar.barStyle = .blackTranslucent
     }
     
+    override func viewWillLayoutSubviews() {
+        videoPreviewLayer.connection?.videoOrientation = AVCaptureVideoOrientation(rawValue: UIApplication.shared.statusBarOrientation.rawValue) ?? .portrait
+        videoPreviewLayer.frame = view.layer.bounds
+    }
```

Comment:
Using this method is quite heavy. Also, it's not calling super here. Can we think of a better way to solve this?

Reply:
When the Batch scanning will be done? After I can fix that.